openhub/affiliated_committers.py

```diff
@@ -5,8 +5,14 @@
 
 
 def get_affiliated_committers_data(json_object):
+    """
+    :param json_object: json data of affiliated committers
+    :return: a list of affiliated committers dict
+    """
     data = json_object['response']['result'
                                    ]['affiliated_committers']['affiliator']
+    if type(data) is not list:
```

Comment:
this is a weak assertion.

it should be `if isinstance(data, foo):` so that the reader knows what it is that is being put into the new `list`